scripts/verify-watchdog.ps1

```
@@ -0,0 +1,62 @@
+        $raw = Get-Content -Path $HeartbeatPath -Raw | ConvertFrom-Json
+        $ts = [string]$raw.timestamp
+        if ([string]::IsNullOrWhiteSpace($ts)) { return [int]::MaxValue }
+        $age = (New-TimeSpan -Start ([datetime]$ts) -End (Get-Date)).TotalMilliseconds
+        if ($age -lt 0) { return 0 }
+        return [int][Math]::Floor($age)
+    } catch {
+        return [int]::MaxValue
+    }
+}
+
+function Ensure-WatchdogTaskRunning {
```

PSScriptAnalyzer (1.24.0)

[warning] The cmdlet 'Ensure-WatchdogTaskRunning' uses an unapproved verb.

(PSUseApprovedVerbs)